ta/sims/ta_entry.c

```diff
@@ -45,18 +45,16 @@ void TA_DestroyEntryPoint(void)
 }
 
 /* Called each time a session is opened */
-TEE_Result TA_OpenSessionEntryPoint(uint32_t nParamTypes, TEE_Param pParams[4],
-				    void **ppSessionContext)
+TEE_Result TA_OpenSessionEntryPoint(uint32_t param_types, TEE_Param params[4],
+				    void **psession)
```

Comment:
Since you are renaming this, why not keep the names as in the GP spec?
```c
TEE_Result TA_EXPORT TA_OpenSessionEntryPoint(
   uint32_t paramTypes,
   [inout] TEE_Param params[4],
   [out][ctx] void**sessionContext );
```

Reply:
camel case I guess... You prefer the GPCamelCaseWay?

Reply:
No, since this is a file implemented by us, I'd go for our coding style, but use the same names as in the GP spec. i.e. like this, `param_types` , `params` and `session_context`.